pkgs/top-level/dotnet-packages.nix

```diff
@@ -220,6 +220,37 @@ let self = dotnetPackages // overrides; dotnetPackages = with self; {
 
   # SOURCE PACKAGES
 
+  Boogie = buildDotnetPackage rec {
+    baseName = "Boogie";
```

Comment:
Nix package versions should never have hyphens in them (otherwise `builtins.parseDrvName` does not work properly). I've pushed the other changes.